### DISCUSSION TITLE
[Script request] inventaire

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#477] [Request script] [answered]
### Application Name

inventaire

### Website

https://github.com/inventaire/inventaire  & https://inventaire.io/

### Description

a libre collaborative resource mapper powered by open-knowledge, starting with books!

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by tremor021** [comment, marked as answer]
Repo is archived. Will not be possible to do this.
